packages/amesos/src/CMakeLists.txt

```diff
@@ -148,14 +137,6 @@ ENDIF()
 #
 
 IF (${PACKAGE_NAME}_ENABLE_KLU)
-  INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR}/SuiteSparse/AMD/Include)
-  INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR}/SuiteSparse/COLAMD/Include)
-  INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR}/SuiteSparse/UFconfig)
-  INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR}/SuiteSparse/KLU/Include)
-  INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR}/SuiteSparse/BTF/Include)
-  INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR}/SuiteSparse/CAMD/Include)
-  INCLUDE_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR}/SuiteSparse/CCOLAMD/Include)
```

Comment:
@sethrj, why are these includes not needed?

Reply:
No subdirectory `SuiteSparse` exists inside `amesos/src`. I can only imagine that this is zombie code (presuming that `SuiteSparse` got moved from here to `packages/common/auxiliarySoftware`). Since no one's complained about their build missing include directories, I can only assume that these can be removed rather than changed.

Reply:
This, by the way, is in contrast to the other CMakeFiles that I changed, since in those cases the standalone Trilinos build would have been correct (since `CMAKE_SOURCE_DIR == Trilinos_SOURCE_DIR`).